File: problemsets/Codeforces/C++/A721.cpp
/**
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 */
#include <bits/stdc++.h>
using namespace std;
int p, l, k, r[100];
int main() {
	cin.tie(0); cout.tie(0);
	ios::sync_with_stdio(0);
  cin >> l;
	string a; cin >> a;
	a = "W" + a + "W";
	while ((l = a.find("WB", p)) != -1) 
		r[k++] = (p=a.find("BW", l))-l;
	cout << k << "\n";
	for (l = 0; l < k; l++) cout << r[l] << " ";
}

File: problemsets/Codeforces/Python/A721.py
"""
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 """
import re
input()
a=[len(x) for x in re.split('[^B]',input()) if x]
print(len(a))
print(*a)
